Openshift-EE/pipelines/Cassandra/stages/4-chaos/S193-jiva-controller-network-delay/jiva-controller-network-delay

```diff
@@ -87,7 +87,7 @@ current_time=$(eval $time)
 # Update the e2e cr once the job is completed
```

Comment:
@w3aman can you include the sed command to append the run_id.

Reply:
Done